### ISSUE TITLE
Peds Smart Templates

### ISSUE THREAD

**Issue by DavidAlt** [DavidAlt/MHSG#17]
Document Smart Templates with peds applications
* what info they display
* where they are used (in)correctly
* how they can be used better

**Comment by absolutmagnitud** [comment]
Request for Newborn Smart Template:
Hi guys-
I am working a ticket requesting the vitals in the newborn discharge be updated to include head circumference, birth weight, discharge weight and length.  I also think HR, RR, Temp and SpO2 could reasonably be included.

Currently, the Smart template (Newborn Vitals and Weight) is pulling the following items from the event set hierarchy:

1: Pulse - L Femoral
2: Pulse - R Femoral
3: Pulse - Bilateral femoral
4: Tone
5: "Neuro Assessment" -not sure what this actually contains

I confirmed that this is the code set being pulled for the smart template.  This is obviously inadequate.

Bottom line, If you were to write a Smart template for newborn vitals what elements would you include?  Am I missing any.  Is what is listed above too much?

Liam - any chance you could get a deidentified copy of a recent newborn discharge for us to see?  I imagine providers are having to hand jam a lot of the info.  Tom and I are going to collaborate on Friday and potentially get this nailed down.  

Thanks,

Andrew Holdaway

**Comment by DavidAlt** [comment]
* Birth weight
* Birth length
* Birth head circumference
* Discharge weight
* Discharge length
* Discharge head circumference
* WGA <-- personal preference, would be nice to see consolidated with vitals (if this will only be used for newborn discharge), but info is generally available elsewhere
* TCB @ 24 HOL! (if only one), otherwise want to see bilirubin levels with date/timestamps (and date/time of birth)
* HR
* RR
* SpO2 (?pre/post-ductal?)
* Temp
